Legion/VaultOfTheWardens/TirathonSaltheril.lua

```diff
@@ -48,17 +52,32 @@ end
 
 function mod:DarkstrikesCast(args)
 	self:Message(191941, "Important", self:Tank() and "Alarm", CL.casting:format(args.spellName))
-	self:Bar(191941, 7, CL.cast:format(args.spellName))
-	self:CDBar(191941, 29)
+	self:CDBar(191941, 31)
 end
 
 function mod:DarkstrikesApplied(args)
 	self:TargetMessage(args.spellId, args.destName, "Urgent")
 	self:Bar(args.spellId, 10, CL.onboss:format(args.spellName))
 end
 
-function mod:Metamorphosis(args)
-	self:Message(192504, "Neutral", "Info")
+function mod:Havoc(args)
+	self:Message(args.spellId, "Neutral", "Info")
+	self:StopBar(191941) --Vengeance Darkstrikes
+	self:CDBar(191941, 24) -- Darkstrikes

```

Comment:
Your intention here is unclear, you are stopping a bar that you are starting straight after, the comments are wrong.